yarn-project/end-to-end/src/e2e_logs.test.ts

```diff
@@ -1,4 +1,4 @@
-import { type AccountWalletWithSecretKey, Fr, TaggedNote } from '@aztec/aztec.js';
+import { type AccountWalletWithSecretKey, type AztecNode, Fr, L1EventPayload, TaggedLog } from '@aztec/aztec.js';
```

Comment:
Would likely be better if the test was named event log or something. Log is ambiguous.